Makefile

```diff
@@ -35,12 +35,22 @@ DOCKER_COMMAND    = docker run --rm -i
 DOCKER_USER       = -u "$(shell id -u):$(shell id -g)"
 DOCKER_VOLUME     = -v "$(shell pwd):/data" -w "/data"
 DOCKER_TEX_VOLUME = -v "$(dir $(realpath $<)):/data" -w "/data"
-
-PANDOC        = $(DOCKER_COMMAND) $(DOCKER_VOLUME)     $(DOCKER_USER) --entrypoint="pandoc"                $(DOCKER_IMAGE)
-HUGO          = $(DOCKER_COMMAND) $(DOCKER_VOLUME)     $(DOCKER_USER) --entrypoint="hugo"                  $(DOCKER_IMAGE)
-DOT           = $(DOCKER_COMMAND) $(DOCKER_VOLUME)     $(DOCKER_USER) --entrypoint="dot"                   $(DOCKER_IMAGE)
-LATEX         = $(DOCKER_COMMAND) $(DOCKER_TEX_VOLUME) $(DOCKER_USER) --entrypoint="latex"                 $(DOCKER_IMAGE)
-DELETE_SCRIPT = $(DOCKER_COMMAND) $(DOCKER_VOLUME)     $(DOCKER_USER) --entrypoint="/opt/delete-script.rb" $(DOCKER_IMAGE)
+# GIT_DIR ensures that git works with the repository
+# no matter the owning user of the directory.
+# see https://github.com/Compilerbau/CB-Lecture-Bachelor/pull/16 for the discussion
+# around this specific workaround and
+# https://github.blog/2022-04-12-git-security-vulnerability-announced/ &
+# https://stackoverflow.com/questions/71901632/fatal-error-unsafe-repository-home-repon-is-owned-by-someone-else
+# for a general overview of the issue.
+#
+# ***Important***: keep the location of GIT_DIR in sync with the mountpoint of the repository inside the container.
+DOCKER_GIT_ENV = --env GIT_DIR=/data/.git
```

Comment:
```suggestion
DOCKER_GIT_ENV = --env GIT_DIR="/data/.git"
```

Reply:
1. Ich habe das jetzt einfach gemacht, schaden tut es nicht. 
<details>
<summary>Insofern unnötige Begründung</summary>
Sollte ja, einfach schon damit einheitlich. Muss aus meiner Sicht (im Moment) nicht, weil `/data` keinerlei Leerzeichen enthält. Das ist insbesondere mit den lokalen Pfaden in Z. 37/38 anders, da jeder das Repo bei sich sonst wo liegen haben kann. Und da greift dann halt die ganze Thematik mit Leerzeichen und Kommandozeilen-Argumenten in Shells.
</details>
2. Ist jetzt in einer Variable :)

Reply:
👍